en/setup/dev_env_linux.md

```diff
@@ -1,123 +1,18 @@
 # Development Environment on Linux
```

Comment:
FYI Top level script has just become index to other scripts rather than being script instructions that linked to the "manual" version in the lower level ubuntu page.